src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.html

```diff
@@ -1,12 +1,54 @@
 <cd-orchestrator-doc-panel *ngIf="showDocPanel"></cd-orchestrator-doc-panel>
-<cd-table *ngIf="hasOrchestrator"
-          #daemonsTable
-          [data]="daemons"
-          [columns]="columns"
-          columnMode="flex"
-          [autoReload]="5000"
-          (fetchData)="getDaemons($event)">
-</cd-table>
+
+<div  *ngIf="flag === 'hostDetails'; else elseBlock">
+  <cd-table *ngIf="hasOrchestrator"
+            #daemonsTable
+            [data]="daemons"
+            [columns]="columns"
+            columnMode="flex"
+            [autoReload]="5000"
+            (fetchData)="getDaemons($event)">
+  </cd-table>
+</div>
+
+<ng-template #elseBlock>
```

Comment:
IMHO "elseBlock" is an undescriptive name for the template, maybe sth like "serviceDetailsTpl" would be better.